### DISCUSSION TITLE
rnote constantly crashes

### DISCUSSION THREAD

**Discussion by Acisama** [flxzt/rnote#1157] [Q&A]
Sometimes when switching between drawing mode and selecting mode, rnote crashes

**Comment by flxzt** [comment]
Please provide the necessary details to debug the issue. See the bug report template.
There are already reports about recurring crashes in #1131 or #1061 so if one of the issues fit your situation, please add a comment with your machine details and a log there instead of opening a new issue

**Reply by Acisama** [reply]
It only has this ouput `
(rnote:31856): Gtk-WARNING **: 10:41:25.760: Theme parser error: gtk.css:5683:3-9: No property named "height"
Segmentation fault (core dumped)`

And I tried it with GSK_RENDERER=gl and I also tried the flatpak version

**Reply by Doublonmousse** [reply]
Do you have a custom css theme ?

**Reply by Acisama** [reply]
No, I wouldn't even know how to do that for rnote

**Reply by flxzt** [reply]
can you please make a bit more effort and provide more info? And post it in the issue (sounds similar to #1131), not here in 'Discussions'.
